aim/web/ui/src/services/models/explorer/createAppModel.ts

```diff
@@ -340,6 +340,7 @@ function createAppModel(appConfig: IAppInitialConfig) {
           config.table = {
             resizeMode: ResizeModeEnum.Resizable,
             rowHeight: RowHeightSize.md,
+            hideSystemMetrics: true,
```

Comment:
let's add this config state also for Metrics table

Reply:
@KaroMourad Metrics table haven't system metrics

Reply:
In Metrics table the Hide/Show button isn't rendered

Reply:
@arsengit the thing is we can have the same table config for all of them to easily integrate for every app model, we need it for app model unification @roubkar  what do you think ?

Reply:
we can think of this during base explorer implementation, for now this is ok I think